pyrevitlib/rpw/ui/forms/resources.py

```diff
@@ -7,9 +7,12 @@
 
 from abc import ABCMeta
 
-from rpw import revit
-from rpw.utils.dotnet import clr
-from rpw.utils.logger import logger
+from pyrevit import revit
```

Comment:
Is this import needed? Since it is not used anywhere in the module, I would drop it to avoid conflicts between the two libraries